apps/api/src/lib/utils.ts

```diff
@@ -35,3 +38,55 @@ export const flattenDayStringsAndSplit = (value: TransformInput): TransformOutpu
         ),
       )
     : undefined;
+
+export function normalizeCourse(course: PrismaCourse): Course {
+  let courseLevel: CourseLevel;
+  switch (course.courseLevel as PrismaCourseLevel) {
+    case "LowerDiv":
+      courseLevel = "Lower Division (1-99)";
+      break;
+    case "UpperDiv":
+      courseLevel = "Upper Division (100-199)";
+      break;
+    case "Graduate":
+      courseLevel = "Graduate/Professional Only (200+)";
+      break;
+  }
+  const geList = (course.geList as Array<Omit<GE, "ANY">>).map((x): GECategory => {
+    switch (x) {
+      case "GE-1A":
+        return "GE Ia: Lower Division Writing";
+      case "GE-1B":
+        return "GE Ib: Upper Division Writing";
+      case "GE-2":
+        return "GE II: Science and Technology";
+      case "GE-3":
+        return "GE III: Social & Behavioral Sciences";
+      case "GE-4":
+        return "GE IV: Arts and Humanities";
+      case "GE-5A":
+        return "GE Va: Quantitative Literacy";
+      case "GE-5B":
+        return "GE Vb: Formal Reasoning";
+      case "GE-6":
+        return "GE VI: Language Other Than English";
+      case "GE-7":
+        return "GE VII: Multicultural Studies";
+      case "GE-8":
+        return "GE VIII: International/Global Issues";
+      // this branch should never happen
+      default:
+        throw new Error();
+    }
```

Comment:
This is technically out of scope of the PR, but what do you think of something like this?

```suggestion
   // At the top of the file somewhere
   const GE_CATEGORIES = {
     'GE-1A': 'GE 1a: Lower Division Writing',
     'GE-1B': 'GE 1b: Upper Division Writing',
     // ...
   }
   
    const category = GE_CATEGORIES[x]
    
    if (category = null) {
      throw new Error()
    }
    
    return category
```

Reply:
Yeah this should definitely have been a lookup table to begin with. I am not really sure what was going on inside my head when I wrote this code :upside_down_face: